src/main/java/com/fasterxml/jackson/databind/DeserializationContext.java

```diff
@@ -254,6 +254,7 @@ protected DeserializationContext(DeserializationContext src) {
         _readCapabilities = src._readCapabilities;
         _view = src._view;
         _injectableValues = null;
+        _currentType = null;
```

Comment:
But why here? None of constructors set it. So let's leave it `null` by default. It is state that is only set and cleared in one place

Reply:
Well, it's more a practical view, trying to be consistent. But there's difference between configuration settings (which I think ctor should set/copy), and then sort of state for reuse.